doc/getting-started.md

```diff
@@ -0,0 +1,29 @@
+# Getting started
+
+## Launch Diun with the Docker provider
+
+Create a `docker-compose.yml` file that uses the official Diun image:
+
+```yaml
+version: "3.5"
+
+services:
+  diun:
+    image: crazymax/diun:latest
+    volumes:
+      - "./data:/data"
+      - "/var/run/docker.sock:/var/run/docker.sock"
+    environment:
+      - "TZ=Europe/Paris"
+      - "LOG_LEVEL=info"
+      - "LOG_JSON=false"
+      - "DIUN_WATCH_WORKERS=20"
+      - "DIUN_WATCH_SCHEDULE=*/30 * * * *"
+      - "DIUN_PROVIDERS_DOCKER=true"
+      - "DIUN_PROVIDERS_DOCKER_WATCHBYDEFAULT=true"
+    restart: always
+```
+
+Here we use a minimal configuration to analyze **all running containers** (watch by default enable) of your **local Docker** instance **every 30 minutes**.
```

Comment:
I get the point of why we want a minimal example here without a lot of overwhelming configuration. What do you think about linking the configurations and providers section to guide the reader what to read next?

Reply:
@Tooa

> I get the point of why we want a minimal example here without a lot of overwhelming configuration.

Yes, that's the idea!

> What do you think about linking the configurations and providers section here to guide the reader what to read next?

I have in mind to make a better integrated documentation through a website. I'll probably use mkdocs-material.